src/app/Plans/components/Wizard/helpers.tsx

```diff
@@ -691,6 +705,20 @@ export const useEditingPlanPrefillEffect = (
   };
 };
 
+export const getClonedPlanDefaultName = (
+  planBeingPrefilled: IPlan,
+  plansQuery: UseQueryResult<IKubeList<IPlan>>
+) => {
+  let name = `Copy of ${planBeingPrefilled.metadata.name}`;
```

Comment:
The problem with the `Copy of...(N)` pattern is that it's not compatible with RFC 1123 DNS Subdomain specification.
Wouldn't it be possible to add `-N` at the end of the source plan name, instead?

Reply:
Ah damn, I forgot about that 😆 my own validation could have told me. Will fix.